src/CattyTests/PlayerEngine/Sensors/Date/DateMonthSensorTest.swift

```diff
@@ -69,8 +69,8 @@ final class DateMonthSensorTest: XCTestCase {
     }
 
     func testConvertToStandardized() {
-        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1))
-        XCTAssertEqual(10, sensor.convertToStandardized(rawValue: 10))
+        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1, landscapeMode: false))
+        XCTAssertEqual(10, sensor.convertToStandardized(rawValue: 10, landscapeMode: false))
```

Comment:
Please also add a test for `convertToStandardized` when called with `landscapeMode` `true`.

Reply:
Done!